taxcalc/params.json

```diff
@@ -920,9 +920,9 @@
     },
     "_NIIT_trt":{
         "long_name": "Net Investment Income Tax rate (UIMC)",
-	    "description": "AKA Unearned Income Medicare Distribution Tax. Taxed on net investment income of individuals with MAGI higher than certain thresholds.",
-        "irs_ref": "Form , line , ",
-        "notes": "HC",
+	"description": "Also called Unearned Income Medicare Contribution Tax. You will be taxed at this rate if your MAGI is higher than NIIT threshold.",
```

Comment:
Just checked, it's on the less of the two amounts. This is from the IRS website.
` individual taxpayers are liable for a 3.8 percent Net Investment Income Tax on the lesser of their net investment income, or the amount by which their modified adjusted gross income exceeds the statutory threshold amount based on their filing status.`
Should we include the entire comparison in the description?